ldms/src/sampler/aries_mmr/aries_lbw.c

```diff
@@ -0,0 +1,598 @@
+/* -*- c-basic-offset: 8 -*-
+ * Copyright (c) 2015-2017 National Technology & Engineering Solutions
+ * of Sandia, LLC (NTESS). Under the terms of Contract DE-NA0003525 with
+ * NTESS, the U.S. Government retains certain rights in this software.
+ * Copyright (c) 2015-2017 Open Grid Computing, Inc. All rights reserved.
```

Comment:
I think OGC does not have a copyright to this code. @tom95858 could you please verify?

Reply:
Same as above, lots of samplers have this.

Reply:
correct, not to my knowledge